Src/Pages/CurrentProvider/CurrentProvider.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useContext, useState } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { StackScreenProps } from '@react-navigation/stack';
+import {
+  NeuroAccessBackground,
+  NavigationHeader,
+  TextLabel,
+  TextLabelVariants,
+  ActionButton,
+  ShowError,
+} from '@Controls/index';
+import { useTranslation } from 'react-i18next';
+import { ThemeContext } from '@Theme/Provider/ThemeContext';
+import { Logo } from '@Assets/Svgs';
+import { GlobalStyle as styles, CurrentProviderStyle } from '@Pages/Styles';
+import { CreateAccountIcon, ChangeProviderIcon } from '@Assets/Svgs';
+
+interface SelectionProp {
+  createAccount: boolean;
```

Comment:
if variable type is boolean then the name should be more understandable like createAcccount -> isCreateAccountSelected and changeProvider -> isChangeProviderSelected

Reply:
done, change variable names.